pyproject.toml

```diff
@@ -166,28 +166,29 @@ filterwarnings = [
 ]
 
 [tool.coverage.run]
-source = ["*/src/spyglass/*"]
-omit = [ # which submodules have no tests
-    "*/__init__.py",
-    "*/_version.py",
-    # "*/common/*",
-    "*/data_import/*",
-    "*/decoding/v0/*",
-    # "*/decoding/*",
-    "*/figurl_views/*",
-    # "*/lfp/*",
-    # "*/linearization/*",
-    "*/lock/*",
-    "*/mua/*",
-    # "*/position/*",
-    "*/ripple/*",
-    "*/sharing/*",
-    "*/spikesorting/v0/*",
-    # "*/spikesorting/*",
-    # "*/utils/*",
-    "settings.py",
-    "*/moseq/*",
-]
+source = ["*/src/spyglass/spikesorting/*burst*"]
+# include = [ "*/src/spyglass/spikesorting/*burst*" ]
```

Comment:
Why are these commented out?

Reply:
Whoops - I didn't mean to push my edits to pyproject. I'll revert these